Run7/calib.cfg

```diff
@@ -0,0 +1,21 @@
+# We need to compute e/pixel/sec given the LED and the current
+[WBLED]
+source = 90
+red    = 1.
+blue   = 1. 
+uv     = 1.
+nm750  = 1.
+nm850  = 1.
+nm960  = 1.
+
+[WBCURRENT]
+uv	= 219.42	-0.45
+
+blue	= 4831.42	0
+
+red	= 5213.16	-5.556
+
+nm750	= 2247.32	0
+
```

Comment:
Looking at the PRs for the other steps, I see that this config file appears in all of them, but the content of this version is missing the `nm850` entry.  If this file should be the same for all steps, it should only appear in one PR (either this one or a separate PR just for this file).